roqoqo-qryd/src/tweezer_devices.rs

```diff
@@ -861,9 +885,17 @@ impl TweezerDevice {
 
     #[inline]
     fn get_current_layout_info(&self) -> &TweezerLayoutInfo {
-        self.layout_register
-            .get(&self.current_layout)
-            .expect("Unexpectedly did not find current layout. Bug in roqoqo-qryd.")
+        if let Some(current) = &self.current_layout {
+            self.layout_register
+                .get(current)
+                .expect("Unexpectedly did not find current layout. Bug in roqoqo-qryd.")
+        } else {
+            // Err(RoqoqoBackendError::GenericError {
+            //     msg: "Tried to access current layout info but no current layout is set."
+            //         .to_string(),
+            // })
+            panic!("Tried to access current layout info but no current layout is set.")
+        }
```

Comment:
Since this actually can happen due to user input, I think we should return Err instead of panic.